parachain/pallets/verifier-lightclient/src/lib.rs

```diff
@@ -143,8 +143,44 @@ decl_module! {
 		#[weight = 0]
 		pub fn import_header(origin, header: EthereumHeader, proof: Vec<EthashProofData>) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
-			Self::validate_header_to_import(&header, &proof)?;
-			Self::import_validated_header(&sender, &header)
+
+			frame_support::debug::trace!(
```

Comment:
Can we import `frame_support::debug` so that we don't have to specificy the full path everytime?